activerecord/test/cases/relation/where_test.rb

```diff
@@ -96,6 +96,18 @@ def test_polymorphic_nested_array_where
       assert_equal expected.to_sql, actual.to_sql
     end
 
+    def test_polymorphic_empty_array_where
+      treasure = Treasure.new
+      treasure.id = 1
+      hidden = HiddenTreasure.new
+      hidden.id = 2
+
+      expected = PriceEstimate.where("1=0")
```

Comment:
true :) `none` is new to me, but I just cherry-picked the commit, not sure it's worth it to update, amend and trigger another ci run (lazy monday)